Appraisals

```diff
@@ -4,5 +4,12 @@ appraise 'rails-6' do
   gem 'net-smtp'
   gem 'net-pop'
   gem 'net-imap'
-  gem 'rails', '~> 6.0', '< 6.1'
+  gem 'rails', '~> 6.0', '< 7.0'
```

Comment:
🔧  This `'< 7.0'` condition is unnecessary here since we have the `~>` locked to minor versions of Rails 6. Can we remove the extra condition?